src/main/resources/config.sample.yml

```diff
@@ -50,8 +53,10 @@ moneytransfer:
     #the password used, to verify a valid amazon giftcode email (enter password as message at amazon homepage, while buying giftcodes
     parsingPassword: test
 
-    #the mail address to send notification mails to
+    #the mail address to send notification mails about moneytransfer issues to
```

Comment:
not anymore, it has never been. I dont know which mail address the notification package uses. @phimonk